### PR TITLE
fix(a380x/flight model): Fix stall and auto rotate issues

### LINKED ISSUE #9269: Unrecoverable stall + flight controls frozen during takeoff
### Aircraft Version

Development

### Build info

```json
{
    "built": "2024-10-31T15:54:41+00:00",
    "ref": "refs/tags/v0.12.0",
    "sha": "2fe89113291f2478347dde0dc3228b55dc924f74",
    "actor": "FoxtrotSierra6829",
    "event_name": "manual",
    "pretty_release_name": "stable/v0.12.0",
    "version": "a380x-v0.12.0-rel.2fe8911"
}
```

### Describe the bug

Takeoff at any gross weight and if you pitch up to anything above 10-12 degrees, it will start to pitch up by itself. You cannot stop this behavior as it seems like the flight controls are frozen once you enter the amber speed protection range or sometimes it pops up if you exceed 17 degrees pitch up. 

Because the flight controls are frozen. you cannot use standard stall recovery to save yourself. The plane will not respond to nose down inputs nor roll inputs. 

The stall at low altitude just means you have no option but to crash. 

### Expected behavior

Expected behavior is before a stall, your flight controls should still respond to the inputs as there is enough airflow going past them  given your airspeed isn't terribly low. The plane should pitch down and start to gain airspeed but that does not happen. 

### Steps to reproduce

1. Takeoff at any gross weight 
2. Pitch up to 13-15 degrees and observe 
3. The plane will keep pitching up by itself and eventually put you in a stall. 

### References (optional)

image

### Additional info (optional)

I have found a workaround for this, given you have enough altitude to recover from the insane nose up attitude. 
1. Full throttle
2. Turn off PRIM 1
3. Turn off SEC 1
4. Turn off PRIM 3
5. Turn off SEC 3
6. Pitch down

I am not a master at Airbus theory, but the turned off Flight control computers should drop you down to alternate law. This change to alternate law seems to reset the elevator, in the sense that it recovers from its jammed pitch state, and the aircraft will respond to any nose down inputs that you give it. After you have recovered and gained enough airspeed, you can turn all the flight computers back on. 

### Discord Username (optional)

banjigaming99

### PR DESCRIPTION
**Pull request by donstim**
Fixes #9269 

## Summary of Changes
1. Increases elevator authority and reduces pitch-up from engine thrust to : a) allow the envelope protection system to perform its intended function of preventing the airplane from stalling and b) allow the pilot to pitch the nose down for recovery.
2. Decreases the effectiveness of the trim system at takeoff speed conditions to prevent auto-rotation
3. Adjusts stable condition pitch angles to better match available data
## References

[Pitch angles and N1.zip](https://github.com/user-attachments/files/17613975/Pitch.angles.and.N1.zip)

## Additional context
Example takeoff showing no autorotation and stall prevention/recovery capability:
https://1drv.ms/v/s!Ajsy7Ey4_8uFgZZ7YVz0k8S3PINZ4Q?e=bjlT6m

Discord username (if different from GitHub): donbikes

## Testing instructions
1. Test takeoffs at high, low, and medium weights with pitch trim properly set and takeoff speeds from SimBrief to verify no auto-rotation. (Note:  At very low weights, the nosewheel may become unweighted just before Vr.)
2. Test going to high pitch angles/low speeds in all configurations to verify that stall is prevented and verify that recovery can be made by reducing the pitch angle by use of the elevator.
3. Test landings at low, medium, and high weights to verify that the THS does not reach its upward limit (10 degrees) during the approach/landing.

## How to download the PR for QA

Every new commit to this PR will cause new A32NX and A380X artifacts to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, find and click on the **PR Build** tab
1. Click on either **flybywire-aircraft-a320-neo** or **flybywire-aircraft-a380-842** download link at the bottom of the page

### CODE REVIEW

File: .github/CHANGELOG.md [hunk 1]
```diff
@@ -15,8 +15,9 @@
 1. [A32NX/CAMERA] Improved default camera position for Virtual Reality (VR) - @aguther (Andreas Guther)
 1. [A380X/EFB] Fixed doors automatically opening in flight - @saschl (saschl)
 1. [A380X/FMS] Fixed layouting issue on FMS/ACTIVE/PERF/T.O page for some users - @flogross89 (floridude)
-1. [A380X/TELEX] Added popup for telex consent - @saschl (saschl) @Maximilian-Reuter (\_chaoz)
+1. [A380X/TELEX] Added popup for telex consent @saschl (saschl) @Maximilian-Reuter (\_chaoz)
```
Comment (2hwk) [inline .github/CHANGELOG.md:18]: Diff shouldn't exist here

**Review by 2hwk — APPROVED** [review]

**Review by 2hwk — CHANGES_REQUESTED** [review]
See changelog.md

**Review by 2hwk — APPROVED** [review]

**Comment by Saschl** [comment]
Quality Assurance Report

Discord Username  : saschl
Aircraft          : A380X
Tier of Testing   : 2
Date              : 04/11/2024

Testing Process: 

Full flight EDDB - EGLL

- [ ] Takeoff and observe nose wheel. Could not observe premature lifting of nosewheel. 
- [ ] Pitched up after takeoff and could always recover
- [ ] Triggered A/FLOOR situation and could recover
- [ ] Checked pitch angles during normal flight, seemed reasonable to me

Negatives:
N/A, great job!

Testing Results: 
Passed

File: fbw-a380x/src/base/flybywire-aircraft-a380-842/SimObjects/AirPlanes/FlyByWire_A380_842/flight_model.cfg [hunk 2]
```diff
@@ -1,26 +1,6 @@
...
-
-; 15 Feb 2024 - Activate new spring method and update strut springs
-
-; 19 Jul 2024 - Adjust CG, wheels and fuel tanks to correct model origin to Airbus reference datum offset
-; The position of the nose in the model coord. system is 35.1927m. The Airbus reference datum distance from the nose is 7.33m.
-; So, the conversion from HArm value to model coordinates in m is: model_coord = (35.1927m + 7.33m) - harm_value
-
-; 13 Sept 2024 fixed some invalid entries in the trigger section
-
-; 22 Oct 2024 Fixed numerous issues with the fuel system. Code added to address MSFS fuel system issues and restrict transfers until airplane is in flight.
```
Comment (Benjozork) [inline fbw-a380x/src/base/flybywire-aircraft-a380-842/SimObjects/AirPlanes/FlyByWire_A380_842/flight_model.cfg:22]: Was this meant to be deleted?

Comment (donstim) [reply fbw-a380x/src/base/flybywire-aircraft-a380-842/SimObjects/AirPlanes/FlyByWire_A380_842/flight_model.cfg:22]: Yes. All this was good for development, but I feel superfluous now. All future modifications will be documented in their respective PRs.

File: .github/CHANGELOG.md [hunk 3]
```diff
@@ -18,7 +18,7 @@
 1. [A380X/TELEX] Added popup for telex consent - @saschl (saschl) @Maximilian-Reuter (\_chaoz)
 1. [ND] Fix memory leak when using TERR ON ND - @Nufflee (nufflee)
 1. [A380X/OVHD] Fix RCDR GND CTL button/logic - @flogross89 (floridude)
-1. [A380X] Various fixes in FMS and ECL @flogross89 (floridude)
+1. [A380X] Various fixes in FMS and ECL - @flogross89 (floridude)
```
Comment (donstim) [inline .github/CHANGELOG.md:22]: I assume this is how it should be?

**Comment by outermarkerXXVII** [comment]
Have you checked the THS during approach and landing? I did a few landings at 360 tons, 35%CG and landed with a THS of ~3% so nearly FULL UP trim.

**Comment by donstim** [comment]
> Have you checked the THS during approach and landing? I did a few landings at 360 tons, 35%CG and landed with a THS of ~3% so nearly FULL UP trim.

Thanks for the report. This will be fixed.

**Comment by aadee9940** [comment]
So i was testing this out yesterday, did a few stalls and from that point of view everything was good. But then when i transitioned to normal flight, autothrottle just did not respond. 

I had autothrottle on, ap1 was not engaged. I selected speed 220 and put the levers into clb. 

I was at 240 knots and expected the engines to drop thrust so i can slow down, instead what happens is as soon as I put the levers into clb, the engines go into toga regardless of what the speed target is. 

The only way to get out of this is was to disconnect autothrottle completely.

**Comment by flogross89** [comment]
@aadee9940 Thanks for the test. ATHR should not be affected by this PR. Just to confirm, you were in selected speed mode? What did the left column of the FMA show? Should show SPEED and not THR CLB or A FLOOR.

**Comment by aadee9940** [comment]
@flogross89 hi, i loaded it up today with a clean install, seems like the issue is gone. Yesterday, the fma was refusing to show speed, it was stuck on thr clb. I tried it today again, and speed mode engages fine. Even with just athr and no ap1 

Thank you.

**Pull request merged**